lib/constants/enums.ts

```diff
@@ -14,6 +14,10 @@ export enum OrderSide {
   Sell,
 }
 
+export enum RaidenTokens {
```

Comment:
We're storing the added currencies and their swap clients on a database level (Currencies). At runtime `SwapClientManager` associates all the currencies/tokens with their respective clients.

What do you think about [checking if we can get](https://github.com/ExchangeUnion/xud/blob/master/lib/swaps/SwapClientManager.ts#L86) the swap client before placing an order? That way we wouldn't have to keep a hardcoded list in the code and always be up to date even if the currencies change at runtime.

Reply:
Thanks didn't know we had that :)

Reply:
@ImmanuelSegol I think we can just use the existing logic that's already in place:
https://github.com/ExchangeUnion/xud/blob/master/lib/orderbook/OrderBook.ts#L347

We just need to make it happen regardless of the value of `nosanitychecks` and also check the swap client for inbound currency.